### PR TITLE
Add field to include and remove gif

### PR DESCRIPTION
**Pull request by deepakkumardk**
Added a filed as includeGif to handle this issue and minor code cleanup

### CODE REVIEW

File: library/src/main/java/com/github/windsekirun/naraeimagepicker/module/PickerSet.kt [hunk 1]
```diff
@@ -55,29 +58,37 @@ object PickerSet {
...
-    fun getFolderList(): List<com.github.windsekirun.naraeimagepicker.item.FolderItem> {
+    fun getFolderList(): List<FolderItem> {
         return PICTURE_MAP.entries
                 .asSequence()
                 .filter { it.value.isNotEmpty() }
                 .map { it.key to it.value[0] }
-                .map { com.github.windsekirun.naraeimagepicker.item.FolderItem(it.first, it.second.imagePath) }
-                .sortedBy { it.name.toLowerCase() }
+                .map { FolderItem(it.first, it.second.imagePath) }
+                .sortedBy { it.name.toLowerCase(Locale.ENGLISH) }
```
Comment (WindSekirun) [inline library/src/main/java/com/github/windsekirun/naraeimagepicker/module/PickerSet.kt]: Do you have reason to using 'Locale.ENGLISH'? if not, Use 'Locale.getDefault()' will support other conturies.

File: library/src/main/java/com/github/windsekirun/naraeimagepicker/module/PickerSet.kt [hunk 2]
```diff
@@ -55,29 +58,37 @@ object PickerSet {
...
                 val image = photoCursor.getColumnString(PATH_COLUMN)
                 val id = photoCursor.getColumnString(ID_COLUMN)
                 val file = image.toFile()
-                if (file.exists()) list.add(FileItem(id, image))
+                if (file.exists())
+                    if (item.includeGif) {
+                        list.add(FileItem(id, image))
+                    } else
+                        if (file.extension != "gif")
+                            list.add(FileItem(id, image))
```
Comment (WindSekirun) [inline library/src/main/java/com/github/windsekirun/naraeimagepicker/module/PickerSet.kt]: I think I can make it simpler.

```
if (file.exists() && (item.includeGIf || file.extension != "gif")) list.add(FileItem(id, image))
```

**Review by WindSekirun — CHANGES_REQUESTED** [review]
There are some minor improvements, but looks good to me!

**Comment by deepakkumardk** [comment]
Committed the requested changes.

**Review by WindSekirun — APPROVED** [review]

**Pull request merged**